tests/phpunit/unit-tests/forms/class-llms-test-form-handler.php

```diff
@@ -334,7 +334,7 @@ public function test_submit_success() {
 
 
 	/**
-	 * Test submit password change without providing, or with wrong current password
+	 * Test submitting account update without password update
```

Comment:
Now the description's are right but the test function name is still transposed / misleading :-p

Reply:
You're never content.